docs/certifications-cis.md

```diff
@@ -0,0 +1,24 @@
+# CIS Certfications
```

Comment:
In order for this to appear in the documentation site, don't forget to make it available in the mkdocs.yml file in the root of the project